test/e2e/util.go

```diff
@@ -70,9 +70,9 @@ func waitForPodCondition(c *client.Client, ns, podName, desc string, condition p
 		if done {
 			return err
 		}
-		Logf("Waiting for pod %s status to be %q (found %q) (%d secs)", podName, api.PodRunning, pod.Status.Phase, time.Since(start).Seconds())
+		Logf("Waiting for pod %s status to be %q (found %q) (%.2f seconds)", podName, api.PodRunning, pod.Status.Phase, time.Since(start).Seconds())
 	}
-	return fmt.Errorf("gave up waiting for pod %s to be %s after %v", podName, desc, podStartTimeout)
+	return fmt.Errorf("gave up waiting for pod %s to be %s after %.2f seconds", podName, desc, podStartTimeout.Seconds())
```

Comment:
This will always be a whole number, since we're printing the timeout, not elapsed time. I'm fine with `%.2f`, if you feel strongly about seeing the `.00`, but I don't think it's a bug :)